openmc/model/funcs.py

```diff
@@ -110,6 +111,140 @@ def borated_water(boron_ppm, temperature=293., pressure=0.1013, temp_unit='K',
     out.add_s_alpha_beta('c_H_in_H2O')
     return out
 
+def cylindrical_prism(r, height, axis='z', origin=(0., 0., 0.),
+                      boundary_type='transmission',
+                      upper_edge_radius=0.,
+                      lower_edge_radius=0.):
+    """Get a finite cylindrical prism.
```

Comment:
Perhaps more information here about the rounded edges.  From reviewing the code, it seems that this option adds what I would call a fillet at each end.

Reply:
+1 for fillet terminology